contracts/ENSGovernor.sol

```diff
@@ -3,17 +3,18 @@ pragma solidity ^0.8.2;
 
 import "@openzeppelin/contracts/governance/Governor.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorProposalThreshold.sol";
-import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
 
-contract ENSGovernor is Governor, GovernorProposalThreshold, GovernorCountingSimple, GovernorVotes, GovernorVotesQuorumFraction, GovernorTimelockControl {
+import "./ENSCounting.sol";
+
+contract ENSGovernor is Governor, GovernorProposalThreshold, ENSCounting, GovernorVotes, GovernorVotesQuorumFraction, GovernorTimelockControl {
```

Comment:
A few generic questions, put at this position arbitrarily:

* Is there a reason we've opted to stick with a legacy version of the OZ Governor rather than upgrading to the 5.x series?
* Is there a reason we're not including the `GovernorSettings` extension (which would allow the governance parameters to be updated by the DAO without having to upgrade the Governor again)?
* I'm biased here, as the original developer of [Flexible Voting](https://flexiblevoting.com/), but the ENS DAO has discussed adding it in the past, and it's increasingly a standard that DAOs are deploying/upgrading to—note that Flexible Voting is now included in the [main line](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/extensions/GovernorCountingFractional.sol#L29) of the OZ Governor, and has/is being adopted by Arbitrum, Compound, Wormhole, zkSync, Gitcoin, Frax, and others. In particular, it would be critical for enabling things like cross chain voting, which might be of interest to ENS given its multichain plans.